### PR TITLE
Annotating link to Spark PRs page with <a>

### PR DESCRIPTION
**Pull request by EnricoMi**
The link can not be clicked otherwise.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ Anyone can view and comment on active changes here.
 Reviewing others' changes is a good way to learn how the change process works and gain exposure 
 to activity in various parts of the code. You can help by reviewing the changes and asking 
 questions or pointing out issues -- as simple as typos or small issues of style.
-See also https://spark-prs.appspot.com/ for a convenient way to view and filter open PRs.
+See also <a href="https://spark-prs.appspot.com/">https://spark-prs.appspot.com/</a> for a
```
Comment (HyukjinKwon) [inline contributing.md:51]: @EnricoMi, can you generate HTML too accordingly? See https://github.com/apache/spark-website#generating-the-website-html

Comment (EnricoMi) [reply contributing.md:51]: I did, I run jekyll 4.0.0 and it generated a lot of unrelated changes. I will try again with <4.0.0.

Comment (HyukjinKwon) [reply contributing.md:51]: Yeah, it happens due to different versions. You can manually exclude unrelated changes except this line

Comment (EnricoMi) [reply contributing.md:51]: Done

**Review by HyukjinKwon — APPROVED** [review]

File: contributing.md (hunk 1 above)
Comment (srowen) [inline contributing.md:51]: It can use markdown link syntax too like `[..](...)` but it doesn't matter. Seems fine.

**Pull request closed without merging**